src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -474,6 +485,49 @@ internal void AddAsset(List<AssetData> assets, string assetName, string version,
             });
         }
 
+        internal (List<PackageArtifactModel>,
+            List<BlobArtifactModel>) MergeManifests(List<Manifest> parsedManifest)
+        {
+            List<PackageArtifactModel> packageArtifacts = new List<PackageArtifactModel>();
+            List<BlobArtifactModel> blobArtifacts = new List<BlobArtifactModel>();
+
+            for (int i = 0; i < parsedManifest.Count; i++)
+            {
+                Manifest manifest = parsedManifest[i];
+                foreach (var package in manifest.Packages)
+                {
+                    PackageArtifactModel packageArtifact = new PackageArtifactModel()
+                    {
+                        Attributes = new Dictionary<string, string>
+                    {
+                        { "NonShipping", package.NonShipping.ToString().ToLower() },
+                    },
+                        Id = package.Id,
+                        Version = package.Version
+                    };
+
+                    packageArtifacts.Add(packageArtifact);
+                }
+
+                foreach (var blob in manifest.Blobs)
+                {
+                    BlobArtifactModel blobArtifact = new BlobArtifactModel()
+                    {
+                        Attributes = new Dictionary<string, string>
+                    {
+                        { "NonShipping", blob.NonShipping.ToString().ToLower() },
+                        { "Category" , !string.IsNullOrEmpty(blob.Category)? blob.Category.ToString().ToUpper() : "NONE" }
```

Comment:
[nit] Probably best to run this through some formatting. 
```suggestion
                        { "Category" , !string.IsNullOrEmpty(blob.Category) ? blob.Category.ToString().ToUpper() : "NONE" }
```

Reply:
updated this